src/EnvironmentChecker.php

```diff
@@ -172,19 +179,42 @@ public function index()
             $response->setStatusCode($this->errorCode);
         }
 
-        $resultText = $result->customise([
-            'URL'       => Director::absoluteBaseURL(),
-            'Title'     => $this->title,
-            'Name'      => $this->checkSuiteName,
-            'ErrorCode' => $this->errorCode,
-        ])->renderWith(__CLASS__);
+        if (!$this->includeDetails) {
+            // if not including details on the public version, at least include
+            // it within the specific email
+            $emailContent = $result->customise([
+                'URL'       => Director::absoluteBaseURL(),
```

Comment:
You have pretty much the same array being created 3 times here - can you please do this in a more DRY manner?

Reply:
updated now